zeebe/broker/src/test/java/io/camunda/zeebe/broker/exporter/stream/ExporterRule.java

```diff
@@ -107,6 +108,13 @@ public void startExporterDirector(final List<ExporterDescriptor> exporterDescrip
 
   public void startExporterDirector(
       final List<ExporterDescriptor> exporterDescriptors, final ExporterPhase phase) {
+    startExporterDirector(exporterDescriptors, phase, Function.identity());
+  }
+
+  public void startExporterDirector(
+      final List<ExporterDescriptor> exporterDescriptors,
+      final ExporterPhase phase,
+      final Function<RecordExporter, RecordExporter> recordExporter) {
```

Comment:
❓ But here we could for simplicity simply accept the record exporter? As in tests it would be mostly an mock? Which we can then use to replace the real one? Wdyt?

Reply:
My idea was that we could just spy on the real instance, so instead of building a mock you can just spy on a method for example.